### DISCUSSION TITLE
Welcome to core Discussions!

### DISCUSSION THREAD

**Discussion by alexstandiford** [berlindb/core#86] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bosconian-dynamics** [comment]
👋 👋 Hello hello!

I think anyone who's worked with WordPress for a good amount of time eventually gets to the point of lamenting storing all of a site's content in `wp_posts`. I was stoked to watch @JJJ's 2019 WCEU presentation and hear all of those frustrations so eloquently phrased in the introduction, and elated that he went on to describe a project which seemed to very cleanly address the issue - and prospectively, maybe one day introduce it's sorely needed utility into core :)

I largely originally learned to code on WordPress many years ago, and COVID times have mandated my return to the platform. I'm seeking to really get involved in more collaborative efforts - and even finally contributing to core and Gutenberg as I am able - both out of excitement to contribute to projects I really ideologically care about as well as to build up a more employable GitHub history. I love the role WordPress plays in facilitating people to express themselves, and feel it's FOSS status really embodies the promise of the internet.

I have always learned best by doing and teaching others, and I'm really looking forward to build a better understanding of BerlinDB through the same approach - it is something which I am very excited to begin using in various projects moving forward 😄

Cheers 🥂

**Reply by JJJ** [reply]
👋 hey @bosconian-dynamics, welcome, and I'm so glad you enjoyed my presentation.

You and I are very much aligned when it comes to FOSS, WordPress, learning, teaching, and cheers'ing. 🥂

Glad that you're back to WordPress, say nothing of the circumstances of the world. 💫

**Comment by JJJ** [comment]
It's been a few years of emptiness, so I'm just gonna close this one 🙃
